modules/oc-mirror-IDMS-ITMS-about.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-custom-resources-v2_{context}"]
+= About custom resources generated by V2
```

Comment:
Some places use capital V2 and others are v2. Should be consistent. Also, shouldn't just say v2 here (or anywhere really), should say the oc-mirror plugin v2

Reply:
Yeah, I have used oc-mirror plugin v2 in most of the places, I will thoroughly check again for Capital v2.